Document suggested setup of package maintained by group.

voxik opened the pull request:
This is attempt to provide more insight how to setup the package ACL when it should be maintained by FAS group.

pypingou commented on doc/groups.rst:
Typo on the last word here: presmissions :)

pypingou commented on doc/groups.rst:
Do we double-check this? I believe it is true but I might have been wrong

voxik commented:
1. It seems I was not right about the "default CC", so I dropped the note.
2. I fixed some quotes.
3. I added another bullet suggesting to cleanup the ACLs, since users don't need to be listed anywhere, if they are already member of some group with appropriate rights.

Let me know if I should squash the commits.

pypingou commented:
Looks all good to me, thanks! :)
